content/services/planning.md

```diff
@@ -161,6 +168,19 @@ summary: |
   with interactive wireframes,
   prioritized estimates,
   and a robust plan for development and launch.
+
+  Using an object-oriented user experience strategy
+  (OOUX), we'll guide you through a
+  process to define the concepts or 
+  "objects" that make up your web app, 
+  the relationships between those objects, 
+  and the ways users want to interact with those objects.
+  We will deliver an Object Map
+  to streamline communication 
+  between designers, developers, and stakeholders 
+  throughout the project.
+
+  [certified object-oriented user experience strategist]: /authors/sondra/
```

Comment:
This makes the summary very long. Consider moving it into the main body content?